src/Blockcore.Indexer.Core/Storage/Mongo/MongoDb.cs

```diff
@@ -55,6 +55,14 @@ public IMongoCollection<InputTable> InputTable
          }
       }
 
+      public IMongoCollection<UtxoTable> UtxoTable
+      {
+         get
+         {
+            return mongoDatabase.GetCollection<UtxoTable>("Utxo");
```

Comment:
So we have both a collection named "Transaction" and this introduces"Utxo"? How about "UnspentTransaction" instead?

Reply:
Well we have output so I'll add unspend Outputs